components/NewEntryButton/NewEntryButton.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {
+  Text,
+  StyleSheet,
+  Pressable,
+  GestureResponderEvent,
+} from "react-native";
+import componentStyles from "./NewEntryButtonStyles";
+
+
+interface INewEntryButton {
+  onPress: (event: GestureResponderEvent) => void;
+  title: string;
+}
+
+export default function NewEntryButton(props: INewEntryButton) {
+  const { onPress, title = "Save" } = props;
```

Comment:
you're defining two new variables so should we give them each a type? I know we know what type props is but at a glance it helps readability. thoughts?